test/mlir/conversion/onnx_to_torch/leaky_relu/leakyalpha_one.onnx.mlir

```diff
@@ -0,0 +1,10 @@
+//RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
+module attributes {llvm.data_layout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128", llvm.target_triple = "x86_64-unknown-linux-gnu"}  {
```

Comment:
NIT: the `attributes ...` do not seem necessary for minimal test cases. (Applies to more tests here.)

Reply:
Liam,
We have removed all the unnecessary attributes since they are not required.